src/data/roadmap/releases.tsx

```diff
@@ -167,7 +170,7 @@ export const releasesData: Release[] = [
         </ul>
       </div>
     ),
-    href: "/roadmap/fusaka",
+    href: "https://eips.ethereum.org/EIPS/eip-7594",
```

Comment:
```suggestion
    href: "https://eips.ethereum.org/EIPS/eip-7607",
```

Thanks for catching this. Main change im making here is just to the hardfork EIP for Fusaka as a whole :D